ios/Classes/SwiftWorkmanagerPlugin.swift

```diff
@@ -109,17 +113,17 @@ extension SwiftWorkmanagerPlugin {
 
 private extension SwiftWorkmanagerPlugin {
     
-    static var callbackHandleStorageKey: String {
-        return "callBackHandleStorageKey"
+    private var callbackHandleStorageKey: String {
```

Comment:
Couldn't this be a const (static let)?

Reply:
Nope, `let` declarations cannot be computed properties